pytket/docs/circuit_class.rst

```diff
@@ -327,6 +327,12 @@ condition on a specified set of bit values.)
 
    .. automethod:: ZZMax
 
+   .. automethod:: AAMS
+
+   .. automethod:: GPI
+
+   .. automethod:: GPI2
```

Comment:
Doesn't have to be in this PR, but maybe these should be in alphabetical order to make them easier to find?

Reply:
The order I opted for was to have similar gates grouped together rather than alphabetical

e.g.

```
   .. automethod:: CRy

   .. automethod:: CRz

   .. automethod:: CU1

   .. automethod:: CU3
   
  maybe alphabetical is better, not sure.